### PR TITLE
Fix a bug where the file size would increase after cropping an image

### PR DESCRIPTION
**Pull request by Nesci28**
fixes the issue 73: 
https://github.com/vugar005/ngx-awesome-uploader/issues/73#issue-917558357

### CODE REVIEW

File: projects/file-picker/package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "Angular Library for uploading files with Real-Time Progress bar, File Preview, Drag && Drop and Custom Template with Multi Language support",
   "peerDependencies": {
     "@angular/common": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0",
-    "@angular/core": " ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0"
+    "@angular/core": " ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0",
+    "mrmime": "^1.0.0"
```
Comment (vugar005) [inline projects/file-picker/package.json:8]: 1) Let's move this package to `"dependencies" `so users don't have to install this package manually.
2) Let's remove ^ caret to install safe and exact version.

**Comment by vugar005** [comment]
Hi @Nesci28 , I added small comment. Thank you for contribution and PR)

File: projects/file-picker/package.json (hunk 1 above)
Comment (Nesci28) [reply projects/file-picker/package.json]: i pushed a commit with the required fixes

**Comment by vugar005** [comment]
@Nesci28  Thanks. merged)

**Pull request merged**